GANDLF/utils/gandlf_logging.py

```diff
@@ -26,6 +28,16 @@ def _configure_logging_with_logfile(log_file, config_path):
         logging.config.dictConfig(config_dict)
 
 
+def gandlf_excepthook(exctype, value, tb):
+    if issubclass(exctype, AssertionError):
```

Comment:
Despite of the initial issue is only about Assertions, let's log absolutely any uncaught exceptions here?

Reply:
Should the check be the following, instead?

```python
if issubclass(exctype, Exception)
```

Reply:
Yes, If we want to log all the uncaught exceptions.

Reply:
Great, then let's make that change and get ready to merge! 😄